src/OrchardCore.Modules/OrchardCore.Deployment.Remote/Controllers/ExportRemoteInstanceController.cs

```diff
@@ -35,13 +34,15 @@ public class ExportRemoteInstanceController : Controller
             RemoteInstanceService service,
             IDeploymentManager deploymentManager,
             INotifier notifier,
+            IHttpClientFactory httpClientFactory,
             IHtmlLocalizer<ExportRemoteInstanceController> localizer)
         {
             _authorizationService = authorizationService;
             _deploymentManager = deploymentManager;
             _session = session;
             _service = service;
             _notifier = notifier;
+            _httpClient = httpClientFactory.CreateClient();
```

Comment:
Don't create it in the constructor.  Remove the global variable and create one before calling it in the action method.

Reply:
I thought about that, but it's fine to move it when it's needed to create the instance